src/lint.ts

```diff
@@ -44,6 +44,13 @@ export interface LintResult {
     output: string;
 }
 
+export interface ILinterOptionsRaw {
```

Comment:
Mixed feelings about if we really need a new interface here - I'm sort of inclined to just making the properties of `ILinterOptions` optional. What do you think?